app/policies/user_policy.rb

```diff
@@ -65,4 +65,11 @@ def change_password?
 
     false
   end
+
+  def can_access_user_account_tab?
```

Comment:
You can use the existing method `staff_can_access_user_account_tab` from `app/policies/application_policy.rb` instead of adding a new method in user policy.